src/dataflow/internal/process/functions/source.ts

```diff
@@ -32,6 +33,11 @@ export function isSourceCall(name: Identifier, scope: DataflowScopeName, environ
 export function processSourceCall<OtherInfo>(functionCall: RFunctionCall<OtherInfo & ParentInformation>, data: DataflowProcessorInformation<OtherInfo & ParentInformation>, information: DataflowInformation): DataflowInformation {
 	const sourceFile = functionCall.arguments[0] as RArgument<ParentInformation> | undefined
 	if(sourceFile?.value?.type == RType.String) {
+		if(getConfig().ignoreSourceCalls) {
```

Comment:
i am still unsure if i would want the check to happen as the first thing in this processor. Especially because we may add behavior for non-constant-string arguments with abstract interpretation etc. and to me it appears to be cleaner if it is a general guard